index.ts

```diff
@@ -233,42 +234,42 @@ export function previous(player: PlayerName, options: any) {
     }
 }
 
-/**
+/** review name
  * Turn on repeat for a given player
- * @param player {spotify|spotify-web|itunes}
+ * @param player {spotify|spotify-web|itunes }
  * @param options
  */
 export function repeatOn(player: PlayerName) {
     return musicCtr.run(player, "repeatOn");
 }
 
-/**
+/** review name
  * Turn off repeat for a given player
- * @param player {spotify|spotify-web|itunes}
+ * @param player {spotify|spotify-web|itunes }
  */
 export function repeatOff(player: PlayerName) {
     return musicCtr.run(player, "repeatOff");
 }
 
-/**
+/** review name
  * Turn on shuffline for a given player
- * @param player {spotify|spotify-web|itunes}
+ * @param player {spotify|spotify-web|itunes }
  */
 export function setShufflingOn(player: PlayerName) {
     return musicCtr.run(player, "setShuffling", ["true"]);
 }
```

Comment:
You might consider making the shuffling functions similar to repeat functions (e.g. pick either repeatOn and shufflingOn ~or~ setRepeatOn and setShufflingOn)